vpr/src/pack/cluster.cpp

```diff
@@ -216,6 +218,10 @@ static enum e_block_pack_status try_place_atom_block_rec(const t_pb_graph_node*
                                                          int verbosity,
                                                          const int feasible_block_array_size);
 
+static enum e_block_pack_status intersect_atom_cluster_part_regions(const AtomBlockId blk_id,
```

Comment:
verbosity should have const too

Reply:
look at clustering context - if variables that don't change are passed all around, then you can pass around verbosity too. but if that is not the case, if they are grabbing control parameters from the clustering context then you can get verbosity from there too instead of passing it around to a bunch of different functions